firebase-appdistribution/src/main/java/com/google/firebase/appdistribution/impl/FirebaseAppDistributionNotificationsManager.java

```diff
@@ -15,25 +15,53 @@
 package com.google.firebase.appdistribution.impl;
 
 import android.app.NotificationChannel;
+import android.app.NotificationChannelGroup;
 import android.app.NotificationManager;
 import android.app.PendingIntent;
 import android.content.Context;
 import android.content.Intent;
+import android.content.pm.ApplicationInfo;
+import android.content.pm.PackageManager;
 import android.os.Build;
+import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
+import androidx.annotation.RequiresApi;
 import androidx.annotation.VisibleForTesting;
 import androidx.core.app.NotificationCompat;
+import androidx.core.app.NotificationManagerCompat;
 
 class FirebaseAppDistributionNotificationsManager {
-  private static final String TAG = "NotificationsManager:";
-  private static final String NOTIFICATION_CHANNEL_ID =
-      "com.google.firebase.appdistribution.notification_channel_id";
+  private static final String TAG = "FirebaseAppDistributionNotificationsManager";
 
   @VisibleForTesting
-  static final String NOTIFICATION_TAG = "com.google.firebase.appdistribution.tag";
+  static final String CHANNEL_GROUP_ID =
+      "com.google.firebase.appdistribution.notification_channel_group_id";
+
+  @VisibleForTesting
+  enum Notification {
+    APP_UPDATE(
+        "com.google.firebase.appdistribution.notification_channel_id",
```

Comment:
maybe remove to common prefix of all Strings (i.e. "com.google.firebase.appdistribution.") and add in in the constructor implementation

Reply:
Yeah I thought about that but wondered if it would be better for search-ability to have the entire string defined here...

But thinking about it again the part after the last `.` is unique enough that that should be quite searchable. Done.